src/pretix/control/forms/filter.py

```diff
@@ -10,7 +10,8 @@
 
 
 class FilterForm(forms.Form):
-    orders = {}
+    orders = {'code': 'code', 'email': 'email', 'total': 'total', 'datetime':
+            'datetime', 'status': 'status', 'pcnt': 'pcnt'}
```

Comment:
This sets it for *all* filter forms, you just need to set this on the specific form, e.g. this one on ``EventOrderFilterForm`` and with the additional entry ``'event': 'event'`` on ``OrderSearchFilterForm``.

Also, watch the indentation of the second line, better format it like:
```
orders = {'code': 'code', 'email': 'email', 'total': 'total', 
          'datetime': ', 'status': 'status', 'pcnt': 'pcnt'}
```